superset/common/query_context.py

```diff
@@ -308,47 +436,16 @@ def get_annotation_data(self, query_obj: QueryObject) -> Dict[str, Any]:
             )
         return annotation_data
 
-    def get_df_payload(  # pylint: disable=too-many-statements,too-many-locals
+    def get_df_payload(
         self, query_obj: QueryObject, force_cached: Optional[bool] = False,
     ) -> Dict[str, Any]:
         """Handles caching around the df payload retrieval"""
         cache_key = self.query_cache_key(query_obj)
-        logger.info("Cache key: %s", cache_key)
-        is_loaded = False
-        stacktrace = None
-        df = pd.DataFrame()
-        cache_value = None
-        status = None
-        query = ""
-        annotation_data = {}
-        error_message = None
-        if cache_key and cache_manager.data_cache and not self.force:
-            cache_value = cache_manager.data_cache.get(cache_key)
-            if cache_value:
-                stats_logger.incr("loading_from_cache")
-                try:
-                    df = cache_value["df"]
-                    query = cache_value["query"]
-                    annotation_data = cache_value.get("annotation_data", {})
-                    status = QueryStatus.SUCCESS
-                    is_loaded = True
-                    stats_logger.incr("loaded_from_cache")
-                except KeyError as ex:
-                    logger.exception(ex)
-                    logger.error(
-                        "Error reading cache: %s",
-                        error_msg_from_exception(ex),
-                        exc_info=True,
-                    )
-                logger.info("Serving from cache")
-
-        if force_cached and not is_loaded:
-            logger.warning(
-                "force_cached (QueryContext): value not found for key %s", cache_key
-            )
-            raise CacheLoadError("Error loading data from cache")
+        _cache = QueryCacheManager.get(
+            cache_key, CacheRegion.DATA, self.force, force_cached,
+        )
```

Comment:
Same question here about naming of `_cache`

Reply:
same before